adapt-service-without-webapi/knotx-standalone-1.1.1.logback.xml

```diff
@@ -0,0 +1,16 @@
+<!--
```

Comment:
Please update README file and verify all cases: https://github.com/Knotx/knotx-tutorials/tree/master/adapt-service-without-webapi